dashboard/app/models/lesson.rb

```diff
@@ -96,19 +96,11 @@ def self.add_lessons(script, lesson_group, raw_lessons, counters, new_suffix, ed
       lesson.reload
 
       Lesson.prevent_multi_page_assessment_outside_final_level(lesson)
-      Lesson.prevent_level_in_lesson_multiple_times(script, lesson)
```

Comment:
This check is now redundant because it fails earlier in that case with a mysql unique key violation on `seeding_key` instead. See the corresponding updated test in script_test.rb.

Reply:
Nice simplification! Just to help myself understand this comment better in the future, the unique key violation will be on: https://github.com/code-dot-org/code-dot-org/blob/34807b70d9822439308349f5f716e097f7bbdfd1/dashboard/app/models/script_level.rb#L23